integration-test-groups/mongodb/mongodb-gridfs/src/test/java/org/apache/camel/quarkus/component/mongodb/gridfs/it/MongodbGridfsIT.java

```diff
@@ -17,9 +17,7 @@
 package org.apache.camel.quarkus.component.mongodb.gridfs.it;
 
 import io.quarkus.test.junit.QuarkusIntegrationTest;
-import org.junit.jupiter.api.Disabled;
 
-@Disabled("https://github.com/apache/camel-quarkus/issues/6341")
 @QuarkusIntegrationTest
```

Comment:
#6341 seems not to be fixed ? Or maybe it's fixed in camel and we update camel-main ?

Reply:
removed when based on main